public/app/features/alerting/unified/components/rules/RulesGroup.tsx

```diff
@@ -60,11 +61,28 @@ export const RulesGroup: FC<Props> = React.memo(({ group, namespace }) => {
   }
 
   const actionIcons: React.ReactNode[] = [];
-  if (hasRuler(rulesSource)) {
-    actionIcons.push(<ActionIcon key="edit" icon="pen" tooltip="edit" />);
-  }
+
+  // for grafana, link to folder views
   if (rulesSource === GRAFANA_RULES_SOURCE_NAME) {
-    actionIcons.push(<ActionIcon key="manage-perms" icon="lock" tooltip="manage permissions" />);
+    const rulerRule = group.rules[0]?.rulerRule;
+    const folderUID = rulerRule && isGrafanaRulerRule(rulerRule) && rulerRule.grafana_alert.namespace_uid;
+    if (folderUID) {
+      const baseUrl = `/dashboards/f/${folderUID}/${kbn.slugifyForUrl(namespace.name)}`;
```

Comment:
is this ok? would it somehow be better to fetch folder url from backend, or ask backend team to include it with rules response?

Reply:
You might have to include `config.appSubUrl` like so:
```suggestion
      const baseUrl = `${config.appSubUrl ?? ''}/dashboards/f/${folderUID}/${kbn.slugifyForUrl(namespace.name)}`;
```

Reply:
thanks! I'll go over and see if it's missing for more links as well

- [ ] include `config.appSubUrl`

Reply:
updated all links to include `appSubUrl`